src/style/prototype.css

```diff
@@ -0,0 +1,82 @@
+.container{
+    padding: 9px 26px;
+}
+
+.wrapper{
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+}
+
+header, .banner, .blog, .news{
+    background-color: #D9D9D9;
+}
+
+header{
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+
+header .logo{
+    width: 181px;
+    height: 37px;
+    background-color: #F1F2AE;
+}
+
+header .menu{
+    display: flex;
+    flex-wrap: wrap;
+}
+
+header .menu .menu__element{
+    width: 46px;
+    height: 37px;
+    margin-right: 19px;
+    background-color: #E7EFD6;
+}
+
+header .menu:last-child{
+    margin-right: 0;
+}
+
+.banner{
+    width: 1065px;
```

Comment:
It should be full-width
```suggestion
    width: 100%;
```

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/40695369/206896583-8ff360fc-8594-4957-beec-f81173537ff3.png">